src/libsync/pushnotifications.cpp

```diff
@@ -57,9 +57,12 @@ void PushNotifications::authenticateOnWebSocket()
     const auto username = credentials->user();
     const auto password = credentials->password();
 
+    printf("username: %s\n", username.toStdString().c_str());
```

Comment:
Don't think you intended to commit this.

Reply:
Yes, sorry. This was not intended. I have removed it